packages/gen-ai/bff/build.openshift.sh

```
@@ -36,20 +36,22 @@ REPO_ROOT="$(git rev-parse --show-toplevel)"
 echo "Repository URL: ${REPO_URL}"
 echo "Branch: ${BRANCH}"
-echo "Context Directory: ${CONTEXT_DIR}"
+echo "Context Directory: . (repo root)"
+echo "Dockerfile Path: ${DOCKERFILE_PATH}"
 echo "Application Name: ${APP_NAME}"
-echo "Dockerfile Path: ${DOCKERFILE}"
 echo "--------------------------------"
 
-# Create the application
+# Create the application (context is repo root so pkg/tls is accessible)
 oc new-app ${REPO_URL}#${BRANCH} \
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)